compute/instances/api-cli/snapshot-import-export-feature.mdx

```diff
@@ -0,0 +1,174 @@
+---
+meta:
+  title: Using the snapshot import/export feature with the API
+  description: This page explains how to import/export Instance snapshots using the Scaleway API
+content:
+  h1: Using the snapshot import/export feature with the API
+  paragraph: This page explains how to import/export Instance snapshots using the Scaleway API
+dates:
+  validation: 2022-07-25
+  posted: 2021-07-25
+---
+
+The snapshot transfer feature allows you to export or import Instances snapshots. 
+Snapshots are created using the QCOW2 format and their data is stored in a Scaleway Object Storage Bucket. These images can then be used to create a copy of an Instance in a different AZ.
+
+More information on this file format, and how to use it can be found in the [QEMU documentation](https://www.qemu.org/docs/master/system/images.html#cmdoption-image-formats-arg-qcow2).
```

Comment:
```suggestion
More information on the QCOW2 file format, and how to use it can be found in the [QEMU documentation](https://www.qemu.org/docs/master/system/images.html#cmdoption-image-formats-arg-qcow2).
```